addons/point_of_sale/models/pos_session.py

```diff
@@ -603,7 +603,7 @@ def close_session_from_ui(self, bank_payment_method_diff_pairs=None):
         return {'successful': True}
 
     def post_close_register_message(self):
-        self.message_post(body=_('Closed Register'))
+        self.message_post(body=_('Closed Register'), email_from=self.env.user.email or "admin@example.com")
```

Comment:
```suggestion
        self.message_post(body=_('Closed Register'))
```

Instead, conditionally call this method.

```py
...
        if self.env.user.email:
            self.post_close_register_message()
...
```